src/calculator.js

```diff
@@ -1,11 +1,5 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+    }
```

Comment:
Parsing error: Unexpected token }

###### See http://eslint.org/docs/rules/ for details about this rule